### PR TITLE
Add SimulatedStressDataset for generating raw synthetic EDA data usin…

### PR DESCRIPTION
**Pull request by Dam4n12**
**Contributors:**  
Shuban Kagini, Weng Shian Ho
skagini2, wsho3  

**Type:** Dataset Contribution

**Description:**  
This PR adds a synthetic dataset for stress classification using simulated electrodermal activity (EDA). It replicates the behavior of `raw_sim_data.csv` from our reproducibility project:  
**Simulation of Health Time Series with Nonstationarity**

EDA signals are generated using NeuroKit2 with different SCR profiles for baseline and stress conditions. Each 10-second window is processed into statistical features (mean, std, min, max) and assigned a binary label (0 = baseline, 1 = stress).

The dataset is implemented as a PyTorch-compatible class and does not require any raw data files or pretrained models.

**Usage:**
```python
from pyhealth.datasets.stress_classification_dataset import SimulatedStressDataset
ds = SimulatedStressDataset()
print(ds[0])

### CODE REVIEW

**Comment by jhnwu3** [comment]
Closing this PR as it lacks proper labeling. Please add appropriate labels and reopen if needed.
